lib/stellar_core_commander/local_process.rb

```diff
@@ -127,7 +127,7 @@ def crash
     private
     def launch_stellar_core
       Dir.chdir @working_dir do
-        @pid = ::Process.spawn("./stellar-core",
+        @pid = ::Process.spawn("./stellar-core run",
```

Comment:
Are you sure this is the syntax we should use?
In other places, it looks like we're using the non-shell version (look for `system` calls) that would look like this:
`::Process.spawn("./stellar-core", "run", :out => "stdout.txt", :err => "stderr.txt")`

Reply:
::Process.spawn supports both syntaxes, I've updated code to use the other one.

Reply:
yeah the new one is not using a shell, so looks good